data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,266 @@
+names(nfl_fandom_google) <- tolower(names(nfl_fandom_google))
+nfl_fandom_google <- nfl_fandom_google %>% 
+  rename(
+    trump_2016_vote = 'trump 2016 vote%'
+  ) %>%
+  mutate(
+    dma = as.factor(dma),
+    nfl = as.numeric(str_replace_all(nfl, "%", "")), 
+    nba = as.numeric(str_replace_all(nba, "%", "")), 
+    mlb = as.numeric(str_replace_all(mlb, "%", "")), 
+    nhl = as.numeric(str_replace_all(nhl, "%", "")), 
+    nascar = as.numeric(str_replace_all(nascar, "%", "")), 
+    cbb = as.numeric(str_replace_all(cbb, "%", "")), 
+    cfb = as.numeric(str_replace_all(cfb, "%", "")), 
+    trump_2016_vote = as.numeric(str_replace_all(trump_2016_vote, "%", ""))
+      )
+#TODO: should I create a sport variable to get rid of nfl, nba etc to make this more long/narrow? How would I do that?
+devtools::use_data(nfl_fandom_google, overwrite = TRUE)
+
+nfl_fandom_surveymonkey <-read_csv("data-raw/nfl-fandom/NFL_fandom_data-surveymonkey.csv", skip=1)
+#TODO: Clarify: unsure of why there are three variables for each race (with different values) for each team-- maybe stats for people who chose that team as first second and third favorite? 
+nfl_fandom_surveymonkey <- clean_names(nfl_fandom_surveymonkey)
+nfl_fandom_surveymonkey <- nfl_fandom_surveymonkey %>%
+  rename(
+    total_respondents = 'tot_respondents',
+    gop_percent = 'goppercent',
+    dem_percent = 'dempercent',
+    ind_percent = 'indpercent',
+    white_percent = 'whitepercent',
+    nonwhite_percent = 'nonwhitepercent'
+  ) %>%
+  mutate(
+    team = as.factor(team),
+    gop_percent = as.numeric(str_replace_all(gop_percent, "%", "")),
+    dem_percent = as.numeric(str_replace_all(dem_percent, "%", "")),
+    ind_percent = as.numeric(str_replace_all(ind_percent, "%", "")),
+    white_percent = as.numeric(str_replace_all(white_percent, "%", "")),
+    nonwhite_percent = as.numeric(str_replace_all(nonwhite_percent, "%", ""))
+  )
+devtools::use_data(nfl_fandom_surveymonkey, overwrite = TRUE)
+
+# puerto-rico-media---------------------------------------------------------------------
+
+google_trends <- read_csv("data-raw/puerto-rico-media/google_trends.csv", skip=2)
+google_trends<-clean_names(google_trends)
+google_trends <- google_trends %>%
+  rename(
+    date = day,
+    hurricane_harvey_us = hurricane_harvey_united_states,
+    hurricane_irma_us = hurricane_irma_united_states,
+    hurricane_maria_us = hurricane_maria_united_states,
+    hurricane_jose_us = hurricane_jose_united_states)
+devtools::use_data(google_trends, overwrite = TRUE)
+
+mediacloud_hurricanes <- read_csv("data-raw/puerto-rico-media/mediacloud_hurricanes.csv")
+mediacloud_hurricanes <- clean_names(mediacloud_hurricanes)
+mediacloud_hurricanes$date <- as.Date(mediacloud_hurricanes$date, format= "%m / %d / %y")
+devtools::use_data(mediacloud_hurricanes, overwrite = TRUE)
+
+mediacloud_states <- read_csv("data-raw/puerto-rico-media/mediacloud_states.csv")
```

Comment:
Add a comment line before line 98 giving brief description of each new data frame you are adding